src/TodoListItem.module.css

```diff
@@ -0,0 +1,34 @@
+.ListItem {
```

Comment:
Class names for module css should be in camelCase.

Please read about the use cases where you can use pascal case:

https://dev.to/sathishskdev/part-1-naming-conventions-the-foundation-of-clean-code-51ng

Reply:
Sorry, I was confused about that also because instructions said "Add a ruleset for class ListItem that changes the text color"

Reply:
It might be a misspelling. 
I will address this issue.

Reply:
Don't start the class name from uppercase letter